OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDiscussionCommentsFragment.java

```diff
@@ -123,9 +127,17 @@ public void onClick(View v) {
                     }
                 });
 
+        EnrolledCoursesResponse courseData = (EnrolledCoursesResponse) getArguments().
```

Comment:
1. [suggestion] make this `final`
2. also, why isn't this parsing happening in the handleExtras() function? 🤔

Reply:
Answering Point 2, 
To limit the `courseDate` scope to this function. What do you think?

Reply:
makes sense!